lib/postprocessing/image.js

```diff
@@ -154,7 +239,12 @@ async function needsImagePostProcess(rendition) {
     await logImageMagickVersion();
 
     // inspect intermediate rendition
-    const metadata = rendition.intermediateMetadata = await getImageMetadata(rendition.path);
+    const metadata = rendition._intermediateMetadata = await getImageMetadata(rendition.path);
```

Comment:
Why not?

Reply:
harder to read for me at least when expecting a list of defined vars that I can skim up/down instead of up/down/right

Reply:
The alternative would be this:

```js
const metadata = await getImageMetadata(rendition.path);
rendition._intermediateMetadata = metadata;
```

At least to me (in this particular case) it seems more hard to read since I have to see first that it's `metadata` on the second line as well...